api/src/main/java/org/semanticweb/owlapi/model/OWLOntologyLoaderConfiguration.java

```diff
@@ -477,4 +489,35 @@ public OWLOntologyLoaderConfiguration setRepairIllegalPunnings(boolean b) {
         configuration.overrides.put(REPAIR_ILLEGAL_PUNNINGS, Boolean.valueOf(b));
         return configuration;
     }
+	
+	/**
+     * @param b
+     *        true if HTTP authorization should be accepted.
+     * @return a copy of this configuration with accepting HTTP authorization set
+     *         to the new value.
+     */
+    @Nonnull
+    public OWLOntologyLoaderConfiguration setAcceptingAuthorization(boolean b) {
```

Comment:
It doesn't make sense to have this value separate from the authorization header value itself. Could you remove this and integrate any functionality that relies on it with ``AUTHORIZATION_VALUE`` instead.

Reply:
I have removed the ACCEPT_HTTP_AUTHORIZATION enum and updated the functionality that will check AuthorizationValue not null or empty before adding the Authorization header